packages/yoroi-ergo-connector/README.md

```diff
@@ -3,3 +3,11 @@
 This experimental extension is the first in a modular design to interface the [Yoroi frontend extension](https://github.com/Emurgo/yoroi-frontend) with dApps. It targets the [Ergo](https://ergoplatform.org/en/) cryptocurrency.
 
 The implementation will follow our [EIP-0012](https://github.com/ergoplatform/eips/pull/23) spec.
+
+### Testing
```

Comment:
When I build it like this I get `SyntaxError: Unexpected token 'export'` when it generates `export var extensionId = ...` in the built project, do I have something set up wrong?